src/carousel/carousel.scss

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../text/index';
+@import '../button/button';
+
+@mixin iui-carousel {
+  @include iui-reset;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+@mixin iui-carousel-slider {
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  &-item {
+    position: absolute;
+    margin-left: auto;
+    margin-right: auto;
+    left: 0;
+    right: 0;
+    flex-shrink: 0;
+    width: 100%;
+    box-sizing: border-box;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: transform $iui-speed-slow ease-in-out;
+    }
+
+    &.iui-previous {
+      transform: translateX(-100%);
+    }
+
+    &.iui-next {
+      transform: translateX(100%);
+    }
+
+    &.iui-current {
+      transform: translateX(0);
+    }
```

Comment:
You know, I'm thinking we might be able to improve this approach by applying transform the parent instead. That way we won't even need to use `position: absolute` and not need to modify every single item.

Reply:
that's correct, except we want to translate in the negative direction. These are the changes I had to make:
![image](https://user-images.githubusercontent.com/9084735/150021520-574ecb18-b03a-43c9-90ce-5d258a90c119.png)
![image](https://user-images.githubusercontent.com/9084735/150021637-263d98ab-700d-4edc-8849-aa8898737202.png)

Reply:
Implemented the code & got the dots to change active state, check it out.